src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+/**
+ * DukeException occurs when commands pertaining to the Duke program are invalid.
+ */
+public class DukeException extends Exception{
+
+    private String error;
+
+    private final static String LINE = "    ____________________________________________________________";
+
+    public DukeException(String error) {
+        super(error);
+        this.error = error;
+    }
+
+    @Override
+    public String toString() {
+        String output = LINE + "\n";
+        if (this.error.equals("todo") || this.error.equals("deadline") || this.error.equals("event")) {
+            output += "     :(  OOPS! The description of a " + this.error + " cannot be empty." + "\n";
```

Comment:
You can define the messages as constants if they are not going to be changed

Reply:
Made changes to 2 error message to define as constants